sympy/algebras/quaternion.py

```diff
@@ -1,7 +1,6 @@
 # References :
 # http://www.euclideanspace.com/maths/algebra/realNormedAlgebra/quaternions/
 # https://en.wikipedia.org/wiki/Quaternion
-from __future__ import print_function
 
 from sympy import S, Rational
```

Comment:
I think that the blank line should also be removed here and in similar cases. It seems that it happens automaticall in some cases but not here because of the comments above.

Reply:
Perhaps an upstream issue to pyupgrade should be opened.

Reply:
@oscarbenjamin Ok, removed it in next commit!